nucypher/core.py

```diff
@@ -296,12 +316,14 @@ def __init__(self,
                  hrac: HRAC,
                  policy_encrypting_key: PublicKey,
                  publisher_verifying_key: PublicKey,
-                 destinations: Dict[ChecksumAddress, EncryptedKeyFrag]):
+                 destinations: Dict[ChecksumAddress, EncryptedKeyFrag],
+                 expiration: int):
         self.threshold = threshold
         self.destinations = destinations
         self.hrac = hrac
         self.policy_encrypting_key = policy_encrypting_key
         self.publisher_verifying_key = publisher_verifying_key
+        self.expiration = expiration
```

Comment:
Do we really need to keep it in the TreasureMap too? I don't think it is used anywhere.